eventtracking/backends/mongodb.py

```diff
@@ -80,13 +80,22 @@ def _create_indexes(self):
         # TODO: The creation of indexes can be moved to a Django
         # management command or equivalent. There is also an option to
         # run the indexing on the background, without locking.
-        self.collection.ensure_index([('time', pymongo.DESCENDING)])
-        self.collection.ensure_index('name')
+        # TODO: Remove pymongo3 backward compatibility code once pymongo4 upgrade is done
+        if pymongo.version_tuple[0] == 3:
```

Comment:
suggestion: Add simple test to make sure its getting the right things depending on versions

Reply:
I have added a pin for pymongo version >=4.0.0
[pymongo pin](https://github.com/openedx/event-tracking/pull/290/files#diff-ffb9a8aa35f8e58dd818a724713720da25a2847e19d36b4bbf5da7d5744415e0R7)
now the legacy code is no longer needed.